designer/client/src/components/sidePanels/ScrollbarsExtended.tsx

```diff
@@ -1,13 +1,49 @@
 /* eslint-disable i18next/no-literal-string */
 import React, { PropsWithChildren, useState, useEffect } from "react";
 import Scrollbars from "react-scrollbars-custom";
-import cn from "classnames";
-import styles from "./ScrollbarsExtended.styl";
+import styled from "@emotion/styled";
+import { Side } from "./SidePanel";
 
 const SCROLLBAR_WIDTH = 40; //some value bigger than real scrollbar width
 const CLEAN_STYLE = null;
+const scrollThumbSize = 8;
```

Comment:
Let's be consistent and make scrollThumbSize and toolbarsGap The same way as CLEAN_STYLE

Reply:
fix